doc/sphinx/language/core/basic.rst

```diff
@@ -227,6 +227,7 @@ rest of the |Coq| manual: :term:`terms <term>` and :term:`types
         | @term_match
         | @term_record
         | @term_generalizing
+        | [| {*; @term } %| @term {? : @type } |] {? @univ_annot }
```

Comment:
What is this supposed to be?

Reply:
Primitive arrays, added by #11604.

Reply:
I see. Probably worth refining in a separate PR then. However, this raises the question whether this PR is actually backportable to `v8.12`. This line should be removed in the backport since it's 8.13-only syntax, but what about the documented features of Ltac2. Is any of what's newly documented in this PR 8.13-specific @JasonGross?

Reply:
If you copy this PR locally onto 8.12 and run 'make doc_gram_rsts', that would identify and partly reverse 8.13 grammar changes.  And then probably a quick review to back out any related textual changes.  I can create that if you like after this is merged.  I guess just make the new PR go against 8.12 instead of master?

Reply:
Thanks, I guess I will try following your advice and backporting myself first. And if I don't manage, I'll ask you to do it.